src/ReachSets/ContinuousPost/BFFPSV18/BFFPSV18.jl

```diff
@@ -24,7 +24,7 @@ end
 function options_BFFPSV18()
     return OptionSpec[
         # general options
-        OptionSpec(:approx_model, "forward", domain=String, domain_check=(
+        OptionSpec(:approximation, "forward", domain=String, domain_check=(
```

Comment:
How about "algorithm"?

Reply:
Both are fine I think.

Reply:
I used the name `:discretization` for the higher level option.

In the inner function `discretize`, i changed the kwarg to `algorithm`, since it just selects the *algorithm* for the approximation model.